tests/test_errors.py

```diff
@@ -100,9 +102,24 @@ def test_raise_output_errors(mapdl, response, expected_error):
         MapdlNote,
         MapdlInfo,
         MapdlVersionError,
+        MapdlInvalidRoutineError,
```

Comment:
```suggestion
        MapdlInvalidRoutineError,
        MapdlCommandIgnoredError,
        MapdlRuntimeError,
```
Is there a particular reason why `MapdlCommandIgnoredError` and `MapdlRuntimeError` are not tested here?

Reply:
Good question....